datahub-kubernetes/datahub/charts/datahub-mae-consumer/README.md

```diff
@@ -31,7 +31,7 @@ Current chart version is `0.2.0`
 | global.hostAliases[0].ip | string | `"192.168.0.104"` |  |
 | image.pullPolicy | string | `"IfNotPresent"` |  |
 | image.repository | string | `"linkedin/datahub-mae-consumer"` |  |
-| image.tag | string | `"latest"` |  |
```

Comment:
seems tedious to keep maintaining these README-s along side code.

Reply:
I agree. But this seems like a standard for helm repos.